SwiftRuntimeLibrary/BaseAssociatedTypeProxy.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using SwiftRuntimeLibrary.SwiftMarshal;
+
+namespace SwiftRuntimeLibrary {
+	namespace ClassWrapTests {
+		[SwiftNativeObject ()]
+		public class BaseAssociatedTypeProxy : ISwiftObject {
+			protected IntPtr handle;
+			protected SwiftMetatype class_handle;
+			protected SwiftObjectFlags object_flags = SwiftObjectFlags.IsSwift;
```

Comment:
Pascal cased or private:

```suggestion
			SwiftObjectFlags object_flags = SwiftObjectFlags.IsSwift;
```